modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -0,0 +1,46 @@
+= Premium upgrade promotion
+:navtitle: {productname} Premium upgrade promotion
+:description: Editor options related to turning the Premium promotion display off
+:keywords: upgrade, promotion, premium, button
+
+== Premum upgrade promotion option
+
+{productname} 6.2 and later includes the `promotion` option. It controls the presentation or otherwise of a Tiny-specific promotion button.
+
+*Type:* `+Boolean+`
+
+*Default value:* `+true+` in Community self-hosted instances; otherwise `+false+`. See xref:#premium-upgrade-promotion-defaults[immediately below] for details
+
+*Possible values:* `+true+`, `+false+`
+
```

Comment:
```suggestion
[[promotion]]

== `+promotion+`

*Type:* `+Boolean+`

*Default value:* `+true+` in Community self-hosted instances; otherwise `+false+`. See xref:#premium-upgrade-promotion-defaults[immediately below] for details

*Possible values:* `+true+`, `+false+`

== Example: Using `+promotion+`

[source,js]
----
tinymce.init({
  selector: "textarea",  // change this value according to your HTML
  promotion: false
});
----
```

Might be a good idea to turn this into a partial and include it here, to make it easier to include it elsewhere. I believe the option should be in `content-appearance.adoc` or `customize-ui.adoc` since they may be the first places users look.

Reply:
@MitchCTiny I noticed this elsewhere in the other doc PRs, but this isn't how you do headings with ids. There should be no blank line between the id and heading. So it should be this instead:
```
[[promotion]]
== `+promotion+`
```

Reply:
Corrected as part of the ‘turn that section into a partial’ push just now.

Reply:
Also, turned that section into a partial and added `include::`s to the suggested pages.

Reply:
Ahh good, I was going to raise it probably should have been a partial but thought I was being too picky.